src/plugins/sync-data-attrs/client.js

```diff
@@ -0,0 +1,18 @@
+var excludedAttrs = [
+	'intlOverrides',
+	'langDefault',
+	'oslo',
+	'timezone'
+];
+
+module.exports = function clientSyncDataAttrs(client) {
+	return client.request('attrs').then(function(dataAttrs) {
+		var htmlElems = document.getElementsByTagName('html');
+		if (htmlElems.length === 1 && dataAttrs && Object.keys(dataAttrs).length > 0) {
+			for (var attrName in dataAttrs) {
+				// Omit data attributes already handled by other plugins
+				if (!excludedAttrs.includes(attrName)) htmlElems[0].dataset[attrName] = dataAttrs[attrName] || '';
```

Comment:
What's the `|| ''` trying to handle? Just worried that if one of them container the string `'false'` that it would evaluate to `false` here and result in the empty string.

Reply:
Ah, this was a relic of when I was setting the attributes a different way. I don't think this is actually needed since `Element.dataset` should always return a map of strings. And you're right that it would catch `'false'` as false-y, which isn't what we want.